tdx-tdcall/src/asm/tdcall.asm

```diff
@@ -19,6 +19,7 @@
 # )
 .global asm_td_call
 asm_td_call:
+        endbr64
```

Comment:
why we need endbr here?

Reply:
It seems global asm functions are called with an indirect CALL/JMP, so the program will crash if we do not add `endbr64` here.

Reply:
Would you please add some reference on that? I am a little surprised on why.

Reply:
This is the disassembly of https://github.com/confidential-containers/td-shim/blob/main/tdx-tdcall/src/lib.rs#L101-L103

```
0000000000000000 <td_call>:
   0:	f3 0f 1e fa          	endbr64 
   4:	48 83 ec 28          	sub    $0x28,%rsp
   8:	48 89 f9             	mov    %rdi,%rcx
   b:	ff 15 00 00 00 00    	call   *0x0(%rip)        # 11 <td_call+0x11>
  11:	48 83 c4 28          	add    $0x28,%rsp
  15:	c3                   	ret
```

Reply:
OK. It is indirect call.